etl/steps/data/garden/wid/2023-01-27/shared.py

```diff
@@ -0,0 +1,269 @@
+    },
+}
+
+# Details for naming each decile/percentile
+pct_dict = {
+    "p0p10": {
+        "decile10": "Poorest decile",
+        "decile9": "Minimum value",
+        "thr_number": 0,
+        "decile10_extra": "Poorest decile (tenth of the population)",
+    },
+    "p10p20": {
+        "decile10": "2nd decile",
+        "decile9": "Poorest decile",
+        "thr_number": 10,
+        "decile10_extra": "2nd decile (tenth of the population)",
+    },
+    "p20p30": {
+        "decile10": "3rd decile",
+        "decile9": "2nd decile",
+        "thr_number": 20,
+        "decile10_extra": "3rd decile (tenth of the population)",
+    },
+    "p30p40": {
+        "decile10": "4th decile",
+        "decile9": "3rd decile",
+        "thr_number": 30,
+        "decile10_extra": "4th decile (tenth of the population)",
+    },
+    "p40p50": {
+        "decile10": "5th decile",
+        "decile9": "4th decile",
+        "thr_number": 40,
+        "decile10_extra": "5th decile (tenth of the population)",
+    },
+    "p50p60": {
+        "decile10": "6th decile",
+        "decile9": "5th decile (median)",
+        "thr_number": 50,
+        "decile10_extra": "6th decile (tenth of the population)",
+    },
+    "p60p70": {
+        "decile10": "7th decile",
+        "decile9": "6th decile",
+        "thr_number": 60,
+        "decile10_extra": "7th decile (tenth of the population)",
+    },
+    "p70p80": {
+        "decile10": "8th decile",
+        "decile9": "7th decile",
+        "thr_number": 70,
+        "decile10_extra": "8th decile (tenth of the population)",
+    },
+    "p80p90": {
+        "decile10": "9th decile",
+        "decile9": "8th decile",
+        "thr_number": 80,
+        "decile10_extra": "9th decile (tenth of the population)",
+    },
+    "p90p100": {
+        "decile10": "Richest decile",
+        "decile9": "Richest decile",
+        "thr_number": 90,
+        "decile10_extra": "Richest decile (tenth of the population)",
+    },
+    "p99p100": {"decile10": "Top 1%", "decile9": "Top 1%", "thr_number": 99, "decile10_extra": "Top 1%"},
+    "p99_9p100": {"decile10": "Top 0.1%", "decile9": "Top 0.1%", "thr_number": 99.9, "decile10_extra": "Top 0.1%"},
+    "p99_99p100": {
+        "decile10": "Top 0.01%",
+        "decile9": "Top 0.01%",
+        "thr_number": 99.99,
+        "decile10_extra": "Top 0.01%",
+    },
+    "p99_999p100": {
+        "decile10": "Top 0.001%",
+        "decile9": "Top 0.001%",
+        "thr_number": 99.999,
+        "decile10_extra": "Top 0.001%",
+    },
+    "p0p50": {"decile10": "Bottom 50%", "decile9": "Bottom 50%", "thr_number": "", "decile10_extra": "Bottom 50%"},
+    "p50p90": {"decile10": "Middle 40%", "decile9": "Middle 40%", "thr_number": "", "decile10_extra": "Middle 40%"},
+}
+
+# To avoid breaking f-strings, I need to add a line break like this
```

Comment:
I understand that you are defining `new_line` just for convenience, in case you decide to change the number of line breaks, right? The explanation sounds as if there's some technical reason why you need to define the line break this way (cause you could simply add `\n\n` inside the f-string and it should be fine, as far as I know).
In any case, given that this is a global constant a the module level, you should name it with capital letters, e.g. `NEW_LINE`.

Reply:
I think it was because at the beginning I was trying to use it inside the brackets in an f-string, which breaks it.

Reply:
Now there's not much need to have it in a variable, so I deleted it. Thanks.